storage/appendblob.go

```diff
@@ -70,7 +70,6 @@ type AppendBlockOptions struct {
 func (b *Blob) AppendBlock(chunk []byte, options *AppendBlockOptions) error {
 	params := url.Values{"comp": {"appendblock"}}
 	headers := b.Container.bsc.client.getStandardHeaders()
-	headers["x-ms-blob-type"] = string(BlobTypeAppend)
```

Comment:
Why does this need to be removed?

Reply:
It's not needed, and while the real service ignores it, it causes problems with the Azurite storage server emulator (see the first linked issue for more details).

Reply:
I'm not sure I understood the whole issue with the user wanting this to be mandatory, but if the service ignores it then this is fine.